pennylane/wires.py

```diff
@@ -31,17 +31,28 @@ class WireError(Exception):
 def _process(wires):
     """Converts the input to a tuple of wire labels.
 
-    If `wires` can be iterated over, its elements are interpreted as wire labels
-    and turned into a tuple. Otherwise, `wires` is interpreted as a single wire label.
+    If ``wires`` can be iterated over, its elements are interpreted as wire labels
+    and converted into a tuple. If ``wires`` is not iterable, it is interpreted as
```

Comment:
```suggestion
    If ``wires`` is iterable, its elements are interpreted as wire labels
    and converted into a tuple. If ``wires`` is not iterable, it is interpreted as
```

Reply:
Do you still want to add these comments @austingmhuang? Based on our conversation I understand erasing the reference to `_process` should be enough to not care about this in this PR for the sake of everyone involved

Reply:
Nope. Let's not change anything in `_process` and `Wires` since I did not touch it when I implemented the set operations. Let's just revert everything :)